src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/StandardLicenceConditions.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.recommendation
+
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.TextValueOption
+
+data class StandardLicenceConditions(
+  val selected: List<String>? = null,
+  val allOptions: List<TextValueOption>? = null
+)
+
+enum class SelectedStandardLicenceConditions {
+  GOOD_BEHAVIOUR,
```

Comment:
I think there are going to be 9 of these; maybe worth checking with Andre

Reply:
Confirmed we are going with 7 for now, we can add the other 2 at a later date